nextest-runner/src/target_runner.rs

```diff
@@ -27,11 +27,11 @@ impl TargetRunner {
     /// or via a `CARGO_TARGET_{TRIPLE}_RUNNER` environment variable
     pub fn new(
         configs: &CargoConfigs,
-        build_platforms: &BuildPlatforms,
```

Comment:
@sunshowers I rename `BuildPlatforms` to `BuildPlatform` as now we decided to not include an array of targets in `BuildPlatform`. WDYT?

Reply:
Ah, I've always imagined the plural in `BuildPlatforms` to consist of "host + target". So I think I'd prefer to keep it that way.